src/sentry/static/sentry/app/components/charts/chartZoom.jsx

```diff
@@ -214,10 +214,12 @@ class ChartZoom extends React.Component {
       return children(props);
     }
 
-    const dateFormat = useShortDate ? 'MMM Do' : 'lll';
+    const dateFormat = useShortDate ? 'MMM Do' : 'MMM D LT';
     const hasShortInterval = useShortInterval(this.props);
     const xAxisOptions = {
       axisLabel: {
+        showMaxLabel: false,
+        showMiLabel: false,
```

Comment:
Should this be showMinLabel?

Reply:
good catch.

Reply:
https://github.com/getsentry/sentry/pull/17356/commits/05fb1c1ace1f3b7b8dc2572850d950645d135347